### PR TITLE
fix: escape special chars in code text

### PR DESCRIPTION
**Pull request by nikophil**
Hello there!

here is a fix for https://github.com/symfony/symfony-docs/pull/17587

I had hard time understanding the problem was only occurring for code of type "text"!

cheers!

### CODE REVIEW

File: src/Renderers/CodeNodeRenderer.php [hunk 1]
```diff
@@ -117,23 +118,6 @@ public static function isLanguageSupported(string $lang): bool
         return \in_array($lang, $supportedLanguages, true);
     }
 
-    private function getLines(string $code): array
```
Comment (nikophil) [inline src/Renderers/CodeNodeRenderer.php:120]: was not used anywhere

**Review by alamirault — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]
Awesome 😎

**Comment by javiereguiluz** [comment]
Very nice fix! Thanks Nicolas.

**Comment by javiereguiluz** [comment]
Thank you @nikophil.

**Pull request merged**